pincer/commands/arg_types.py

```diff
@@ -19,16 +22,22 @@ class CommandArg(metaclass=_CommandTypeMeta):
     """
     Holds the parameters of an application command option
 
+    .. note::
+        Deprecated. :class:`typing.Annotated` or :class:`typing_extensions.Annotated`
```

Comment:
I didn't do this because having Annotated twice and they look like they're from the same module is weird. It also wasn't linking to the python docs properly :skull: